src/ophys_etl/transforms/convert_rois.py

```diff
@@ -147,12 +154,24 @@ def binarize_and_create(self):
             movie_shape = open_vid['data'][0].shape
 
         # binarize the masks
-        self.logger.info("Binarizing the ROIs created by Suite2p.")
+        self.logger.info("Filtering and Binarizing the ROIs created by "
+                         "Suite2p.")
         coo_rois = suite2p_rois_to_coo(suite2p_stats, movie_shape)
 
+        # filter raw rois by linear dimension
+        edge_trsh = self.args['longest_edge_threshold']
+        self.logger.info("Linearly filtering ROIs to remove suite2p "
```

Comment:
Linear filtering means something in image processing. I would use a different term to describe this process of filtering out ROIs by size.

Reply:
Removed term in favor of new terminology